tox/config.py

```diff
@@ -809,10 +840,11 @@ def _list_section_factors(self, section):
                 factors.update(*mapcat(_split_factor_expr, exprs))
         return factors
 
-    def make_envconfig(self, name, section, subs, config, replace=True):
+    def make_envconfig(self, name, section, subs, config, replace=True,
```

Comment:
note that the list here is globally change-able; you probably want to default to None, and if it's none then locally swap to "testenv"